docs/user/dashboard/make-dashboards-interactive.asciidoc

```diff
@@ -659,7 +659,7 @@ Note:
 
 |
 | event.values
-| An array of all cell values for the raw on which the action will execute.
+| An array of all cell values for the raw on which the action will execute. To access a column value, use `{{event.value.[x]}}`, where `x` represents the column number.
```

Comment:
Should this be `event.values.[x]` where `value` is `values`? Same comment on other change

Reply:
Yes — good catch. Updated with 8640ea5f7c3ccddca76b10762c4e490bd95884b9.